lib/pickle/session/parser.rb

```diff
@@ -2,8 +2,18 @@ module Pickle
   module Session
     # add ability to parse model names as fields, using a session
     module Parser
+      module ParseFieldWithModel
+        def parse_field(field)
+          if session && field =~ /^(\w+): #{capture_model}$/
+            {$1 => session.model!($2)}
+          else
+            super(field)
+          end
+        end
+      end
+
       def self.included(parser_class)
-        parser_class.alias_method_chain :parse_field, :model
+        parser_class.prepend ParseFieldWithModel
```

Comment:
public `prepend` requires ruby 2.1, but if I do `.send(:prepend, ...)` then ruby 2.0 is fine